codingstandards/continuousintegration.md

```diff
@@ -0,0 +1,55 @@
+# Database Continuous Integration
+## Overview
+Database Continuous Integration (CI) is a method in which changes to the database are recorded incrementally in order to provide a continuously updated database. This form of integration allows for developers to have local copies of a database and to update their local database incrementally.  
+
+## Install vs. Update
+For each database using the CI design there are two levels of versioning that need to be changed. They are `install` and `update`.
+
+### Install
+The `install` changes are all the changes needed to create the database from scratch. This will include all table declorations and stored procedures. The `install` changes will be applied whenever a devoloper wants to create a database. This reqires the `install` directory to have **the current state of the database**. All changes to the database will modify the `install` directory.
+
+### Update
+The `update` changes are all the changes needed to get the database from point **A** to point **B**. The `update` changes are versioned with a unique modifier so that the database knows what updates it has completed and which ones it needs to run. The PROD database will run the `update` changes whenever a new change is released. 
+
+## Performing a Change
+When making a change to a database using the CI design follow these steps.
+
+### 1. Modify Install directory
+
+Modifiy the `*ProjectName*/Database/Install` directory to reflect the current state of the database. The `Install` directory has multiple subdirectories for the different components of the database. 
+
+![Install folder](/images/LegacyWebInstallFolder.png)
+
+* The `Functions` directory stores functions.
```

Comment:
We usually put functions under procedures, perhaps in a subfolder.  If this isn't what has been practiced, though, it's not a biggie.

Reply:
I just looked at the LegacyWeb code in the packages. But I'll put the functions under procedures for the documentation!